capsul/study_config/config_modules/nipype_config.py

```diff
@@ -1,11 +1,4 @@
 # -*- coding: utf-8 -*-
-##########################################################################
```

Comment:
Hi @sapetnioc, is there a reason why you deleted these licence comments in some files?

Reply:
Well, I'd like to remove them all but I do it only when I edit a file and see it. For me we should not use headers in all files with elements that may vary on time such as date, maintainer, institution name, etc. And for the license, I think a single file is sufficient and much more easy to manage.

Reply:
Okay, I thought this was a mistake or a side-effect of `modernize` so I restored quite a few of these headers in aims.

Maybe we should open a dedicated [issue on brainvisa.github.io](https://github.com/brainvisa/brainvisa.github.io/issues) and discuss that issue there?